lib/bounces.js

```diff
@@ -47,6 +47,19 @@ module.exports = function (log, error) {
         // But we can measure it!
         log.increment('account.email_bounced.already_verified')
       }
+      // Because we have no session token on this endpoint, metrics context
+      // metadata was stashed against a synthesized token for the benefit of
+      // the activity events. This fake request object allows the correct
+      // metadata to be gathered when we emit the events.
+      var fakeRequestObject = {
```

Comment:
(the corollary to this is that we'd also need to change the behaviour of `gather` so that it doesn't clear the data after fetching it, so that it stays available for subsequent events)